pkgs/misc/drivers/sc-controller/default.nix

```diff
@@ -49,11 +49,13 @@ buildPythonApplication rec {
   '';
 
   checkPhase = ''
-    PYTHONPATH=. py.test
+    runHook preCheck
+    PYTHONPATH=$PYTHONPATH:. py.test
```

Comment:
Why not use pytestCheckHook?

Reply:
I'm not familiar with this, what's the difference?

Reply:
that pytestCheckHook allows to easily disable tests without knowing the right boolean syntax for pytest.

Reply:
Done. Learnt a new thing.